docs/configuration/logs.md

```diff
@@ -146,19 +148,26 @@ format = "json"
 
   [accessLog.filters]
 
-  # statusCodes keep access logs with status codes in the specified range
+  # statusCodes: keep access logs with status codes in the specified range
```

Comment:
`statusCodes: keep access logs with status codes within the specified range`

Reply:
that description has not changed as result of this PR, should we block the feature because of it? Honestly asking

Reply:
Ah, sorry, it's early here. You are correct :)

Reply:
I see! :D I was honestly asking, there may be a policy in place in the project to better things as they are spotted, did not know. Thx for your comments!